### DISCUSSION TITLE
Creating no-cache custom nodes

### DISCUSSION THREAD

**Discussion by rhalaly** [Comfy-Org/ComfyUI#4660] [Q&A]
I want to create a no-cache custom nodes that need to re-trigger every time as they have dimension of randomness, something like `KSampler`. How can I do that?

**Comment by ltdrdata** [comment]
https://github.com/ltdrdata/ComfyUI-Impact-Pack/blob/84fbb3fa2e43bda785110c726ef1e74508c6b3b3/modules/impact/impact_pack.py#L1841

Return `float("NaN")` in IS_CHANGED
